exercicios/para-casa/Biblioteca.py

```diff
@@ -0,0 +1,31 @@
+from Livro import Livro
+
+class Biblioteca:
+    def __init__(self):
+        self.livros = []
+    
+    def adicionar_livro(self, livro: Livro):
+        if(not isinstance(livro, Livro)):
+            raise TypeError(f"Esperado Livro obtido valor {livro} do tipo {type(livro)}")
+        self.livros.append(livro)
+        #raise NotImplementedError
+    def exibir_livros(self):
+        return self.livros         
+        #raise NotImplementedError
+
+    def emprestar_livro(self, nome, livro: Livro):
```

Comment:
Oi, Mari. Aqui vc não precisa receber o livro não, ta bem? só o nome é o suficiente, vc não está utilizando esse parametro livro daqui, mas o de dentro do for  :)